lib/noted_web/live/note_live.html.leex

```diff
@@ -32,13 +32,24 @@
     <%= error_tag f, :body %>
 <% end %>
 
-
 <div class="py-1 bg-white rounded-b-md">
     <form phx-submit="upload_images" phx-change="validate_upload">
         <section class="p-6">
             <div class="">
-            <%= for image <- Enum.filter(@note.files, & String.starts_with?(&1.mimetype, "image/")) do %>
-                <img src="<%= Routes.file_path(@socket, :serve, image.id) %>" class="w-full" />
+            <%= for image <- filter_images(@note.files) do %>
```

Comment:
This way the filtering is still happening in the template. I think it should happen in the LiveView and be set as two different assigns, `@files` and `@images`. So the mounted code that loads the note data should also call the filtering functions and set up those assigns.